generator/internal/language/rust.go

```diff
@@ -1035,6 +1038,33 @@ func processParagraph(node ast.Node, links map[string]bool, documentationBytes [
 	return results
 }
 
+func fetchPlaceholders(node ast.Node, documentationBytes []byte) map[string]string {
```

Comment:
placeholders is not a great name.  Maybe `fetchHtmlTags` or `fetchHtmlSubstitutions`?

Reply:
Done, good point.